src/locale/languages/fr.js

```diff
@@ -228,6 +228,7 @@ export const fr = {
     },
     readMe: {
       readMeActions: 'Actions sur le README',
+      noReadMeFound: 'No README.md found',
     },
```

Comment:
French translation: `Pas de README.md trouvé`

Reply:
Thank you @machour 

Also done, please check it out! 👍